news/2025/2025-02-01-featured-artist-track-updates-akiri.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Featured Artist Track Updates: Akiri"
+date: 2024-02-01 19:00:00 +0000
+---
+
+Instead of announcing a new Featured Artist this weekend, we're putting the spotlight on an older one: **Akiri** is back, adding a boatload of new songs to their Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/401/header.jpg)
+
+For anyone not in the know, **Akiri** (a.k.a. [ERA Akiri](https://osu.ppy.sh/users/19587411) here on osu!) is a devout key smasher who [debuted as a Featured Artist](https://osu.ppy.sh/home/news/2024-02-11-new-featured-artist-akiri) during [last year's osu!mania 7K World Cup](/wiki/Tournaments/MWC/2024_7K) with an grandiose [7 minute tiebreaker track](https://www.youtube.com/watch?v=bEHMtQxhzqM)! It's been a year since then, and **Akiri** was eager to bring their newest tunes to osu!, so here we are.
```

Comment:
```suggestion
For anyone not in the know, **Akiri** (a.k.a. [ERA Akiri](https://osu.ppy.sh/users/19587411) here on osu!) is a devout key smasher who [debuted as a Featured Artist](https://osu.ppy.sh/home/news/2024-02-11-new-featured-artist-akiri) during [last year's osu!mania 7K World Cup](/wiki/Tournaments/MWC/2024_7K) with a grandiose [7-minute tiebreaker track](https://www.youtube.com/watch?v=bEHMtQxhzqM)! It's been a year since then, and **Akiri** was eager to bring their newest tunes to osu!, so here we are.
```